custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
-$dark-background: #070721; 
-$highlight-text: #f7f5f9;
```

Comment:
Consider using more descriptive variable names to improve readability and maintainability of the styles. [View Reference](https://github.com/raiaqt)